README.md

```diff
@@ -111,11 +112,15 @@ Go to the details page for one of the failed messages and click the `Edit & retr
 
 Navigate to the `Message Body` tab and change some of the contents of the message, ensuring it's still valid JSON matching the message type, and click "Retry" to schedule the message for reprocessing.
 
-> [!WARNING]
+> ⚠️**Warning**
 > Changing or deleting header values may change or cause issues with the subsequent re-processing. It is recommended that these values are not changed if you are unsure of their purpose.
 
 ![Edit Message View](docs/service-pulse-edit-before-retry.png "Edit & Retry view showing the message body")
 
+## View how this works on youtube
+
+[![YouTube Thumbnail](https://img.youtube.com/vi/h5gcHWizS7o/maxresdefault.jpg)](https://www.youtube.com/watch?v=h5gcHWizS7o&t=3s&ab_channel=ChrisPatterson)
```

Comment:
Would it be possible to get an image with the "red and white play triangle" overlayed on it?

Reply:
thi image I got is trough the youtube API. I can make one myself if you want, but we would have to add it as a resource in the repo. That's what I wanted to avoid

Reply:
Embedding the YouTube video should have a built-in way in GitHub that renders the embedded video, vs using an image.

Reply:
Seems this was followed? 
https://ardalis.com/how-to-embed-youtube-video-in-github-readme-markdown/

Reply:
```suggestion
[![YouTube Thumbnail](https://github.com/user-attachments/assets/73fbdbdf-39c0-4a1e-a818-8e234c9ba714)](https://www.youtube.com/watch?v=h5gcHWizS7o&t=3s&ab_channel=ChrisPatterson)
```

I think it's better enough to have the image here to do it.

Reply:
> thi image I got is trough the youtube API. I can make one myself if you want, but we would have to add it as a resource in the repo. That's what I wanted to avoid

Sort of. Yes, around what is written in there